### PR TITLE
PHPDoc fix

### PR DESCRIPTION
**Pull request by saharin88**
LanguageHelper::parseIniFile return array!

### CODE REVIEW

File: libraries/src/Language/LanguageHelper.php [hunk 1]
```diff
@@ -413,7 +413,7 @@ public static function getContentLanguages($publishedStates = array(1), $checkIn
 	 * @param   string   $fileName  The language ini file path.
 	 * @param   boolean  $debug     If set to true debug language ini file.
 	 *
-	 * @return  boolean  True if saved, false otherwise.
+	 * @return  array
```
Comment (zero-24) [inline libraries/src/Language/LanguageHelper.php:416]:
```suggestion
	 * @return  True if saved, or an empty array on error
```

**Comment by zero-24** [comment]
Hi thanks @saharin88 please check the method and my inline comment. For some reason we return true or an empty array :D

**Comment by saharin88** [comment]
> 
> 
> Hi thanks @saharin88 please check the method and my inline comment. For some reason we return true or an empty array :D

Please show me row, where him return bool? 
зображення

**Comment by zero-24** [comment]
ups .. you are right. Merging

**Comment by zero-24** [comment]
Thanks @saharin88 👍

**Pull request merged**